### ISSUE TITLE
PCNTL constants on Windows breaking my test environment

### ISSUE THREAD

**Issue by linslin** [mcustiel/phiremock-codeception-extension#4]
Hey m8, 

you are using [PCNTL](http://php.net/manual/de/pcntl.constants.php) constants which are not working on Windows. This is breaking my test environment at this moment. 

File: `PhiremockProccess.php`

```
    /**
     * Stops the process.
     */
    public function stop()
    {
        $this->process->signal(SIGTERM);
        $this->process->stop(3, SIGKILL);
    }
```

**Comment by linslin** [comment]
I also fixed an other issue depending on a hardcoded directory separator - https://github.com/mcustiel/phiremock-codeception-extension/pull/5/commits/51cb3d0e7b668cd4abbad3eec2c54cd2e7c136d4

**Comment by mcustiel** [comment]
Hey @linslin. Thank you very much for your help with this issue.

**Comment by mcustiel** [comment]
Fixed in version 1.2.3

**Comment by linslin** [comment]
@mcustiel glad to help! Thank you so far!
